pkg/pullsync/pullsync.go

```diff
@@ -419,15 +414,15 @@ func (s *Syncer) collectAddrs(ctx context.Context, bin uint8, start uint64) ([]*
 					topmost = c.BinID
 				}
 				limit--
-				if timer == nil {
-					timer = time.NewTimer(batchTimeout)
-				} else {
-					if !timer.Stop() {
-						<-timer.C
-					}
-					timer.Reset(batchTimeout)
+				if timerC == nil {
+					timerC = time.After(DefaultBatchTimeout)
```

Comment:
once you get the first chunk, max amount of time to wait is now 15 seconds.
the idea with the longer wait is so that requests can be grouped together more often.

Reply:
Got it, all good then.

Reply:
let's keep in mind though that sync now has a min duration delay for the chunk to propagate in the neighborhood.
if this affects things like pss or other features, we can configure the duration, or change back the behavior to what it was before.